pkg/plugins/common/kustomize/v2/scaffolds/api.go

```diff
@@ -76,8 +75,6 @@ func (s *apiScaffolder) Scaffold() error {
 			&samples.CRDSample{Force: s.force},
 			&rbac.CRDEditorRole{},
 			&rbac.CRDViewerRole{},
-			&patches.EnableWebhookPatch{},
-			&patches.EnableCAInjectionPatch{},
```

Comment:
Should be generated only when we scaffold a webhook
Therefore it was moved to the webhook subcommand